BrainCompanyFilingLanguageMetrics.cs

```diff
@@ -45,7 +45,27 @@ public class BrainCompanyFilingLanguageMetrics
 
         public decimal? SpecificDensity { get; set; }
 
-        public BrainCompanyFilingLanguageMetricsSimilarityDifference Similarity { get; set; }
+        public BrainCompanyFilingLanguageMetricsSimilarityDifference? Similarity { get; set; }
+
+        public static BrainCompanyFilingLanguageMetrics Parse(List<string> metrics)
+        {
+            return new BrainCompanyFilingLanguageMetrics
+            {
+                SentenceCount = !string.IsNullOrWhiteSpace(metrics[0]) ? (int)QuantConnect.Parse.Decimal(metrics[0]) : null,
+                MeanSentenceLength = !string.IsNullOrWhiteSpace(metrics[1]) ? QuantConnect.Parse.Decimal(metrics[1]) : null,
+                Sentiment = !string.IsNullOrWhiteSpace(metrics[2]) ? QuantConnect.Parse.Decimal(metrics[2]) : null,
+                Uncertainty = !string.IsNullOrWhiteSpace(metrics[3]) ? QuantConnect.Parse.Decimal(metrics[3]) : null,
+                Litigious = !string.IsNullOrWhiteSpace(metrics[4]) ? QuantConnect.Parse.Decimal(metrics[4]) : null,
+                Constraining = !string.IsNullOrWhiteSpace(metrics[5]) ? QuantConnect.Parse.Decimal(metrics[5]) : null,
+                Interesting = !string.IsNullOrWhiteSpace(metrics[6]) ? QuantConnect.Parse.Decimal(metrics[6]) : null,
+                Readability = !string.IsNullOrWhiteSpace(metrics[7]) ? QuantConnect.Parse.Decimal(metrics[7]) : null,
+                LexicalRichness = !string.IsNullOrWhiteSpace(metrics[8]) ? QuantConnect.Parse.Decimal(metrics[8]) : null,
+                LexicalDensity = !string.IsNullOrWhiteSpace(metrics[9]) ? QuantConnect.Parse.Decimal(metrics[9]) : null,
+                SpecificDensity = !string.IsNullOrWhiteSpace(metrics[10]) ? QuantConnect.Parse.Decimal(metrics[10]) : null,
+
+                Similarity = null
+            };
+        }
 
         public static BrainCompanyFilingLanguageMetrics Parse(List<string> metrics, List<string> similarity)
```

Comment:
Would it not be better to change this method to:
```csharp
public static BrainCompanyFilingLanguageMetrics Parse(List<string> metrics, List<string> similarity = null)
```
and if we call it without the second parameter, `Similarity` would be `null`? 
It would remove the duplicate code.

At least, this method could call the new one, and only set `Similarity`.